prereq_install.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
you should `set -e` IIRC to exit the script on any non-zero exit code of a command ("stop on error" thing)

Reply:
what about `set -e` here and in other scripts?

Reply:
done